how-to/openldap/install-openldap.md

```diff
@@ -208,18 +234,29 @@ Here we used an LDAP "filter": `(uid=john)`. LDAP filters are very flexible and
 (&(objectClass=posixGroup)(memberUid=john))
 ```
 
+NOTE: Replacing `(uid=john)` with this doesn't actually return anything. A filter that the user could try would be pretty helpful.
```

Comment:
Indeed, the `&(objectClass=posixGroup)(memberUid=john))` doesn't seem to produce any results. I was unable to find another working filter, without any LDAP experience on my end.

@toddobryan Do you have a working example, by chance, or maybe @sergiodj or @panlinux can help with that?